chaincfg/params.go

```diff
@@ -34,6 +34,10 @@ var (
 	// simNetPowLimit is the highest proof of work value a Bitcoin block
 	// can have for the simulation test network.  It is the value 2^255 - 1.
 	simNetPowLimit = new(big.Int).Sub(new(big.Int).Lsh(bigOne, 255), bigOne)
+
+	// segnet4 has a pow limit of 1e01ffff which is 19 0-bits in front, I think
```

Comment:
It's 23 0-bits, but I'd prefer this to be consistent with the other comments:

```Go
	// segnet4PowLimit is the highest proof of work value a Bitcoin block
	// can have for the segregated witness test network.  It is the value
	// 2^233 - 1.
```

The math checks out:  `2^233-1 == 0x000001ffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` which when compacted to the Bits representation is `0x1e01ffff` as expected.